project-base/app/tests/FrontendApiBundle/Functional/Login/LoginTest.php

```diff
@@ -62,9 +62,6 @@ public function testInvalidTokenInHeader()
             'errors' => [
                 [
                     'message' => 'Token is not valid.',
-                    'extensions' => [
```

Comment:
I am not sure how exactly this works but I removed method [TokenAuthenticator::onAuthenticationFailure()](https://github.com/shopsys/shopsys/blob/15.0/project-base/app/src/FrontendApi/Model/Token/TokenAuthenticator.php#L25-L36), and in comment is mentioned when #2387 is merged that override is unnecessary and after running tests, these extensions will not return so I thought this would be that. Was my thoughts right?

Reply:
I had to investigate it further, actually the https://github.com/shopsys/shopsys/pull/2387 was about wrapping the errors into an array, but that line with `userCode` was a part of a different feature that had to be persisted. So I fixed it :wink: 
![image](https://github.com/shopsys/shopsys/assets/10401898/4ac4c3a6-30b6-4f3f-a69b-cda6bf289bcc)